README_dev.md

```diff
@@ -128,14 +130,17 @@ npm run test:local -- testSuiteName
           "platformClientId": "...",
           "platformClientSecret": "..."
       },
-      "database": { "driver": "lokijs", "folder": "./" },
+      "database": { "driver": "lokijs", "folder": "./", "dbName": "local", "dbNamePrefix": "l" },
       "logging": { "categories": { "default": { "appenders": ["console"] } } },
-      "infrastructure": { "httpServer": { "apiKey": "xxx", port: 8080 } },
-      "modules": { "coreHttpApi": { "docs": { "enabled": true } } }
+      "infrastructure": { "httpServer": { "apiKey": "xxx", "port": 8080 } },
+      "modules": { "coreHttpApi": { "docs": { "enabled": true } } },
+      "transportLibrary": { "allowIdentityCreation": true }
```

Comment:
There is already a `transportLibrary` property above. Therefore, the `allowIdentityCreation` boolean must be moved.

This configuration option `allowIdentityCreation` is also new to me. Please make sure that this is briefly described in our documentation (https://enmeshed.eu/operate/configuration#transportlibrary) in another PR.

Reply:
Yea that option is actually a copy pasta error will be coming in another PR